src/main/java/com/cydeo/accountingsimplified/service/implementation/CategoryServiceImpl.java

```diff
@@ -13,19 +15,19 @@
 import java.util.List;
 import java.util.stream.Collectors;
 
+
 @Service
 public class CategoryServiceImpl implements CategoryService {
     private final CategoryRepository categoryRepository;
     private final SecurityService securityService;
+    private final ProductRepository productRepository;
```

Comment:
We should call ProductService not Reporsitory

Reply:
I am going to fix it.